source/grid/grid_tools.cc

```diff
@@ -364,6 +364,22 @@ namespace GridTools
   }
 
 
+
+  template <int dim, int spacedim>
+  BoundingBox<spacedim>
+  compute_bounding_box(const Triangulation<dim, spacedim> &tria)
+  {
+    using iterator = typename Triangulation<dim, spacedim>::active_cell_iterator;
+    const auto predicate = [](const iterator &)
+    {
+      return true;
+    };
+
+    return compute_bounding_box(tria, predicate);
```

Comment:
maybe before returning, in case of `p::d::Tria`, use `Utilities::MPI::min/max` to do union of boxes? Or add assert that `p::d::Tria` is not supported.

Reply:
@bangerth what do you think about `compute_bounding_box` of p::d::Tria?

Reply:
I think I'm ok with the current status. One could add a sentence such as "If the triangulation passed as an argument to this function is a parallel::distributed::Triangulation, then each processor will compute a bounding box that encloses all locally owned, ghost, and artificial cells. In the case of a domain without curved boundaries, these bounding boxes will all agree between processors because the union of areas occupied by artificial and ghost cells equals the union of the areas occupied by the cells owned by other processors. However, if the domain has curved boundaries, that is not the case and the bounding box returned may only be appropriate for the current processor, but may be different from the bounding boxes computed on other processors."

Alternatively, as you suggest, one could take the union over all processors to get a single, globally unique, bounding box.

Reply:
Oh yes, you are right! We are looking at active cells in predicate, not locally owned.